### ISSUE TITLE
Character `-->` is not handled correctly

### ISSUE THREAD

**Issue by pcav** [brechtm/rinohtype#65]
This is because the font used in the Sphinx stylesheet does not contain a glyph for this unicode character. Rinohtype warns about this when rendering:

```
TeXGyrePagella-Regular does not contain glyph for unicode index 0x2023 (‣)
```

**Comment by brechtm** [comment]
This will be fixed by #55.

**Comment by brechtm** [comment]
This is fixed in 0.3.0.
